ci/taos/plugins-staging/pr-nnstreamer-ubuntu-apptest.sh

```diff
@@ -55,6 +51,7 @@ function pr-nnstreamer-ubuntu-apptest-run-queue() {
     check_dependency wget
     check_dependency python
     check_dependency Xvnc
+    check_dependency git
```

Comment:
Looks good.

Reply:
Please append the below statement for insmod command

`check_dependency insmod`